data/org.freedesktop.portal.OpenURI.xml

```diff
@@ -74,6 +74,7 @@
     <method name="OpenURI">
       <arg type="s" name="parent_window" direction="in"/>
       <arg type="s" name="uri" direction="in"/>
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In2" value="QVariantMap"/>
       <arg type="a{sv}" name="options" direction="in"/>
```

Comment:
a{sv} isn't needed to be special cased, which makes up a lot of these changes.

Reply:
We already have a whole bunch of those annotations where `a{sv}` is annotated with a `QVariantMap`. Could we remove all of them?

Reply:
@pterror do you know why the annotations on `a{sv}` exist and why @davidedmundson claims they are unnecessary?